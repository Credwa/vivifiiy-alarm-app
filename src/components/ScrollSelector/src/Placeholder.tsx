import React from 'react';
import { View } from 'react-native';

interface IProps {
  wrapperHeight?: number;
  itemHeight?: number;
}
const PlaceHolder = ({ wrapperHeight, itemHeight }: IProps) => {
  const height = ((wrapperHeight || 180) - (itemHeight || 60)) / 2;
  return <View style={{ height, flex: 1 }} />;
};

export default PlaceHolder;
