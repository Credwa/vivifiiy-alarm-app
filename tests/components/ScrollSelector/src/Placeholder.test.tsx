import React from 'react';
import renderer from 'react-test-renderer';
import Placeholder from '@/components/ScrollSelector/src/Placeholder';

describe('ScrollSelector/src/ <Placeholder />', () => {
  it(`renders correctly`, () => {
    const tree = renderer.create(<Placeholder itemHeight={50} wrapperHeight={300} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
