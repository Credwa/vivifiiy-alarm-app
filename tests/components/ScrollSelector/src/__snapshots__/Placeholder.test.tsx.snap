// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScrollSelector/src/ <Placeholder /> renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "height": 125,
    }
  }
/>
`;
